tools/params_processors/swift_compiler_params_processor_tests.py

```diff
@@ -448,7 +448,6 @@ def _parse_args(args):
                 build_mode = "bazel",
             ),
             [
-                "'-F$(DEVELOPER_DIR)/Hi'",
```

Comment:
Can you add `-F` to the test that is testing `-I`, in all the same variations?

Reply:
Thanks, updated!